soccer/src/soccer/strategy/agent/position/seeker.cpp

```diff
@@ -163,14 +163,20 @@ double Seeker::eval_point(rj_geometry::Point ball_pos, rj_geometry::Point curren
     float min_path_dist = 10000;
     for (auto bot : world_state->their_robots) {
```

Comment:
Remove unnecessary copy:
```suggestion
    for (const RobotState& bot : world_state->their_robots) {
```
or, `const auto&`, or (less standard) `auto&&`